base/applications/mspaint/selectionmodel.cpp

```diff
@@ -540,3 +540,25 @@ void SelectionModel::UnlockBitmap(HBITMAP hbmLocked)
 {
     m_hbmColor = hbmLocked;
 }
+
+void SelectionModel::ControlPlusMinus(BOOL bMinus)
+{
+    if (!m_bShow)
+        return;
+
+    TakeOff();
```

Comment:
What does this function do?

Reply:
The selected area will take off.
(The image of selected area is transported)

Reply:
Readability is improved.